django/core/management/base.py

```diff
@@ -57,6 +58,9 @@ def __init__(
     ):
         self.missing_args_message = missing_args_message
         self.called_from_command_line = called_from_command_line
+        # Enable suggest_on_error for Python 3.14+
+        if PY314 and called_from_command_line:
```

Comment:
I just spotted that the default has flipped to `True` on Python 3.15: https://github.com/python/cpython/commit/d2f3cfd38457204a2c47162de6f89ee3dd22fa99

Therefore, let's change the check to:
```suggestion
        if PY314 and not PY315 and called_from_command_line:
            # Adopt Python 3.15 default early.
```

This way, when we drop Python 3.14 support, it should be clear that we can remove this block.